tests/test_post.py

```diff
@@ -156,6 +156,26 @@ def test_menuinst_validation_fails_bad_json(testing_config, caplog, tmp_path):
     assert "JSONDecodeError" in captured_text
 
 
+def test_file_hash(testing_config, caplog, tmp_path):
+    "3rd check - non-parsable JSON fails validation"
```

Comment:
This docstring is a copy-paste artifact I think. Can you add one for this test? Thanks 🙏

Reply:
```suggestion
    "check that the post-link check caching takes the file path into consideration"
```